resources/styles/variables/colors.less

```diff
@@ -35,20 +35,31 @@
 @exercise-question-uid-color: lighten(@tutor-neutral,20%);
 
 // TODO: Move book styles to react-components
-// NOTE: These must be kept in sync with the books in src/flux/book-appearance-codes.coffee
-@openstax-book-appearance-codes: default physics hs_physics biology principles_economics macro_economics college_physics micro_economics concepts_biology intro_sociology anatomy_physiology;
+// NOTE:
+//  * The 'biology' and 'physics' codes are deprecated, but are retained for older courses that may have them
+//  * These must be kept in sync with the books in src/flux/book-appearance-codes.coffee
+@openstax-book-appearance-codes: default physics hs_physics college_physics biology ap_biology concepts_biology college_biology principles_economics macro_economics micro_economics intro_sociology anatomy_physiology;
+
```

Comment:
would be nice to see if we can use variables from [here](https://github.com/openstax/tutor-js/blob/master/resources/styles/book-content/variables.less), or [here](https://github.com/openstax/tutor-js/blob/master/resources/styles/book-content/ap-bio/variables.less) or [here](https://github.com/openstax/tutor-js/blob/master/resources/styles/book-content/hs-physics/variables.less) or gather all such subject/theme/etc. related variables to one place.